Sources/ScrollingTabController.swift

```diff
@@ -349,15 +262,11 @@ public class ScrollingTabController: UIViewController, UIScrollViewDelegate, UIC
         
         let height = CGRectGetHeight(collectionView.frame)
 
-        if let width = self.dataSource?.tabView(self, widthForCellAtIndex: indexPath.item) {
-            return CGSizeMake(width, height)
-        } else {
-            if let flowLayout = collectionViewLayout as? UICollectionViewFlowLayout {
-                return flowLayout.itemSize
-            }
-            
-            return CGSizeMake(100.0, height)
+        if let flowLayout = collectionViewLayout as? UICollectionViewFlowLayout {
```

Comment:
Maybe leave a delegate for this case? Or should we just handle sizing dynamically?